src/bun.zig

```diff
@@ -3139,15 +3140,15 @@ pub fn NewRefCounted(comptime T: type, comptime deinit_fn: ?fn (self: *T) void)
 
     for (std.meta.fields(T)) |field| {
         if (strings.eqlComptime(field.name, "ref_count")) {
-            if (field.default_value == null) {
+            if (field.default_value_ptr == null) {
                 @compileError("Expected a field named \"ref_count\" with a default value of 1 on " ++ @typeName(T));
             }
         }
     }
 
-    const output_name: []const u8 = if (@hasDecl(T, "DEBUG_REFCOUNT_NAME")) T.DEBUG_REFCOUNT_NAME else meta.typeBaseName(@typeName(T));
-
-    const log = Output.scoped(output_name, true);
+    // const output_name: []const u8 = if (@hasDecl(T, "DEBUG_REFCOUNT_NAME")) T.DEBUG_REFCOUNT_NAME else meta.typeBaseName(@typeName(T));
+    // const log = Output.scoped(output_name, true);
+    const log = Output.scoped("TODO:", true);
```

Comment:
sad this was useful

Reply:
gonna bring it back by adding a optional parameter to NewRefCounted